packages-private/aio-commerce-lib-api/source/lib/commerce/types.ts

```diff
@@ -0,0 +1,61 @@
+import type {
+  ImsAuthProvider,
+  IntegrationAuthParams,
+  IntegrationAuthProvider,
+} from "@adobe/aio-commerce-lib-auth";
+import type { Options } from "ky";
+import type { ImsAuthParamsWithOptionalScopes } from "~/utils/auth/ims-scopes";
+
+type CommerceHttpClientConfigBase = {
+  /** The base URL of the Commerce API. */
+  baseUrl: string;
+
+  /**
+   * The store view code use to make requests to the Commerce API.
+   * @default "default"
+   */
+  storeViewCode?: string;
+
+  /**
+   * The version of the Commerce API to use. Currently only `v1` is supported.
+   * @default "v1"
+   */
+  version?: "v1";
```

Comment:
```suggestion
  version?: "V1";
```

The version will always be `V1` or some combination of `V(number)`.

Reply:
```suggestion
  version?: `V${number};
```
alternative with string interpolation